packages/nameguard-react/src/components/icons/ErrorShield.tsx

```diff
@@ -0,0 +1,25 @@
+import { ShieldSize } from "../Report/Shield";
+import { ErrorShieldLarge } from "./ErrorShieldLarge";
+import { ErrorShieldMedium } from "./ErrorShieldMedium";
+import { ErrorShieldSmall } from "./ErrorShieldSmall";
+import { ErrorShieldMicro } from "./ErrorShieldMicro";
+
+import React from "react";
+
+export const ErrorShield = ({
```

Comment:
I don't believe we should call this an "ErrorShield" or an "ErrorShieldIcon" because the icon doesn't really visually express the idea of an error.

Instead, this icon represents uncertainty. We don't know what the actual rating is.

Suggest renaming this UI component and the ideas related to it to "UnknownShieldIcon".